web/main/views.py

```diff
@@ -2416,7 +2416,7 @@ def search_using(request, source):
     return JsonResponse({"results": results}, status=200)
 
 
-type_param_to_category = {'legal_doc': 'legal_doc', 'casebooks': 'casebook', 'users': 'user', 'legal_doc_fulltext': 'legal_doc_fulltext', 'textblock':'textblock', 'link':'link'}
+type_param_to_category = {'legal_doc': 'legal_doc', 'casebook': 'casebook', 'user': 'user', 'legal_doc_fulltext': 'legal_doc_fulltext', 'textblock':'textblock', 'link':'link'}
```

Comment:
I thought there might be a future-reason to want it, but I see now that it makes sense to just chuck it.

Reply:
Ooops, I'm so sorry: I should have said. It also serves to prevent people from being able to supply arbitrary values for category. So if not a mapping, we need at least a list of permitted values.